cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -223,7 +223,13 @@ open class Node @Inject constructor(private val project: Project) : CordformNode
             config = config.withValue("notary", ConfigValueFactory.fromMap(notary))
         }
         if (extraConfig != null) {
-            config = config.withFallback(ConfigFactory.parseMap(extraConfig))
+            val extraProperties = ConfigFactory.parseMap(extraConfig)
```

Comment:
The original point of these extraProperties was to allow users to add any extra config that we haven't exposed via the DSL. This would push this all under "custom" and break the feature.